Pidaq/Lib/uploadToDrive/drive_upload.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+import os
+
+from googleapiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+
+def uploadCsv(filePath):

```

Comment:
Have you thought about being able to pass the filepath as a command line argument? Maybe what you could have is an `if __name__ == '__main__'` Which passes the first command line argument to the `filePath` parameter, and maybe you could have it so that the client_secret.json file gets passed as a second argument to the function, that way it can also be specified as a command line argument. The reason it would be good to have this feature is so that we can also manually upload files from the command line, instead of just invoking this function in other python files.

We can chat about this today

Reply:
That's a sick idea and would make it a full on tool, but at the same time manual uploads can also be done via google drive.